package-lock.json

```diff
@@ -13,6 +13,7 @@
         "clsx": "^1.1.1"
       },
       "devDependencies": {
+        "@storybook/addon-links": "^7.0.21",
```

Comment:
I think it is better to wait with emerging this PR until [Storybook update PR](https://github.com/livechat/design-system/pull/602) will be merged first, because there we have an updated package-lock.json